components/post_profile_picture/post_profile_picture.tsx

```diff
@@ -119,6 +119,9 @@ export default class PostProfilePicture extends React.PureComponent<Props> {
                 username={user?.username}
                 overwriteIcon={this.props.overwriteIcon}
                 overwriteName={overwriteName}
+                isBot={user && user.is_bot}
```

Comment:
Nit: we could simplify this a bit
```suggestion
                isBot={user?.is_bot}
```